lib/hydra/works/services/generic_file/persist_derivatives_output_file.rb

```diff
@@ -0,0 +1,22 @@
+require 'hydra/derivatives'
+
+module Hydra::Works
+  class PersistDerivativesOutputFile < Hydra::Derivatives::PersistOutputFileService
+
+    # Persists a Derivative to a GenericFile
+    # @param [Hydra::Works::GenericFile::Base] object the file will be added to
+    # @param [File] file the dervivative filestream
+    # @param [String] destination_name path to the file
+    # @option opts [RDF::URI or String] type URI for the RDF.type that identifies the file's role within the generic_file
+    # @option opts [Boolean] update_existing when set to false, always adds a new file.  When set to true, performs a create_or_update
```

Comment:
@flyingzumwalt do we need to allow these flags for AddFileToGenericFile to be overridden in this service method signature? Or should we just select appropriate defaults for derivative files?